web/impact/impact/graphql/query.py

```diff
@@ -7,13 +7,41 @@
 from accelerator.models import (
     EntrepreneurProfile,
     ExpertProfile,
+    UserRole,
 )
-from accelerator_abstract.models.base_user_role import is_finalist_user
 from accelerator_abstract.models.base_user_utils import is_employee
 from graphql import GraphQLError
+
 ENTREPRENEUR_NOT_FOUND_MESSAGE = 'Entrepreneur matching the id does not exist.'
 EXPERT_NOT_FOUND_MESSAGE = 'Expert matching the id does not exist.'
-NON_FINALIST_PROFILE_MESSAGE = 'Sorry, You are not allowed to access this page.'
+NOT_ALLOWED_ACCESS_MESSAGE = 'Sorry, You are not allowed to access this page.'
+
+
+def visible_roles(current_user):
+    basic_user_roles = [
+        UserRole.FINALIST,
+        UserRole.AIR,
+        UserRole.MENTOR,
+        UserRole.PARTNER,
+        UserRole.ALUM
+    ]
+    basic_visible_roles = [UserRole.FINALIST, UserRole.STAFF, UserRole.ALUM]
+
+    current_user_roles = list(
```

Comment:
Why are we making this into a list here?

If we leave it as a queryset, then we do not need to convert it back into a set on line 36

Reply:
I'm making `current_user_roles` a list so that I can perform a set intersection operation on `basic_user_roles`.
I'm converting `basic_user_roles` to set, not `current_user_roles`